crates/oxc_linter/src/utils/jest/parse_jest_fn.rs

```diff
@@ -100,7 +101,8 @@ pub fn parse_jest_fn_call<'a>(
 
         let mut call_chains = Vec::from([Cow::Borrowed(name)]);
         call_chains.extend(members.iter().filter_map(KnownMemberExpressionProperty::name));
-        if !is_valid_jest_call(&call_chains) {
+
+        if !is_valid_jest_call(&call_chains) && !is_valid_vitest_call(&call_chains) {
```

Comment:
I think we only need to check `is_valid_vitest_call` if it's using vitest. You can also use `ctx.frameworks().is_vitest` here.